tests/integration-tests/test_runner.py

```diff
@@ -437,14 +437,16 @@ def _get_pytest_args(args, regions, log_file, out_dir):  # noqa: C901
     pytest_args.append(f"--key-path={args.key_path}")
     pytest_args.extend(["--stackname-suffix", args.stackname_suffix])
 
+    # if args.benchmarks:
```

Comment:
Also curious about this block of comments -- if the block is useful explain why it is here / when to uncomment, otherwise let's remove